src/components/Card/Card.jsx

```diff
@@ -0,0 +1,17 @@
+import './Card.css';
+import getRegionColor from '../../helpers/getRegionColor.js';
+
+export default function Card({ flag, name, population, region }) {
+
+  return (
+    <div className="card-wrapper">
+      <div className="card-header">
+        <img src={flag}
+             alt={`flag of ${name}`}
+             className='country-flag' />
+        <h4 style={{color:getRegionColor(region)}}>{name}</h4>
```

Comment:
Op deze manier hoef je minder classes aan te maken in de CSS, dus ik snap waarom je het zo zou doen. Echter heb gebruik je voor de styling overal een stylesheet en hier opeens inline styling. Consistentie is nog belangrijker, anders wordt het onoverzichtelijk.